help/whats_new/_posts/2019-02-01-whats-new.md

```diff
@@ -3,6 +3,39 @@ This page is updated monthly with a list of new (**[new]**) and improved (**[imp
 
 To receive this listing via monthly email newsletter, along with announcements and new blog posts, [sign up here](http://eepurl.com/ggoxhn).
 
+## May 2024
+* Announcements:
+   + We are extremely pleased to introduce our very first cohort of NASA SciX Ambassadors! These exceptional individuals, representing diverse research disciplines, will play an important role in expanding the reach and impact of NASA SciX by serving as liaisons between the platform and their communities. Through their work, they will enhance the interdisciplinary spirit of NASA SciX and work with us to advance the cause of open science. Learn more about each of the new [NASA SciX Ambassadors here](https://www.scixplorer.org/scixblog/announcing-2024-ambassadors).
+     
+   + NASA SciX team members will attend the [2024 Astrobiology Science Conference](https://www.agu.org/abscicon) (#AbSciCon24) from May 5-10 in Providence, RI. If you’ll be there, we invite you to stop by our booth in the exhibit hall to meet our Astrophysics Project Scientist, [Dr. Jennifer Lynn Bartlett](https://www.scixplorer.org/scixabout/team/team/jbartlett.html), and Data Curation Librarian, [Donna Thompson](https://www.scixplorer.org/scixabout/team/team/dthompson.html). We'll be at booth #5, providing demonstrations of the latest features of the NASA SciX platform. The exhibit hall is open Monday through Wednesday from 9:30–11:30 am and 1–6 pm and we hope you’ll stop by to say hello and explore NASA SciX's newest capabilities. 
```

Comment:
Same as above, change to relative paths using the non-SciX path, like this: `/about/team/team/jbartlett.html`. Update for the two team members here and for Felix in the next paragraph below.